src/puppetlabs/trapperkeeper/services/webserver/jetty10_service.clj

```diff
@@ -49,7 +49,19 @@
                           ;; Here for backward compatibility with existing projects
                           (tk-config/get-in-config config-service [:jetty])
                           {})
+               _ (log/debug (i18n/trs "Jetty server(s) starting with config: {0}" config))
                started-context (core/start! context config)]
+           ;; Log started server(s) configuration(s) for debugging purposes
+           (doseq [started-server-context (:jetty10-servers started-context)]
+             ;; Server context data is in the form {:servername {:handlers <handlers>, ... }}
+             (let [server-name (name (first started-server-context))
+                   started-server (:server (second started-server-context))
```

Comment:
What type is this object?

Reply:
Jetty Server https://eclipse.dev/jetty/javadoc/jetty-10/org/eclipse/jetty/server/Server.html - will add type hint.